<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timezonesConfig</class>
 <widget class="QWidget" name="timezonesConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>264</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="KTreeWidgetSearchLine" name="searchLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="urlDropsEnabled">
      <bool>false</bool>
     </property>
     <property name="clickMessage">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Dtgtzwidget" name="timeZones">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>150</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select one or several time zones.</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Area</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Region</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NATO Time Zone</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NATO Time Zone</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Will ceil the offsets to full hours (e.g. UTC+12:45 becomes UTC+12)</string>
       </property>
       <property name="text">
        <string>Use only full letters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fullLetters">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>Dtgtzwidget</class>
   <extends>QTreeWidget</extends>
   <header>dtgtzwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
